# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../php-ssh2/php-ssh2.conf
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; version 2 of the License. A copy of the
# GNU General Public License can be found in the file COPYING.
# --- SDE-COPYRIGHT-NOTE-END ---

# Source: http://svn.mandriva.com/cgi-bin/viewvc.cgi/packages?view=rev&revision=59940
# Author:Oden Eriksson <oeriksson@mandriva.com>
--- ./ssh2.c	2006-06-07 19:35:34.000000000 +0200
+++ ./ssh2.c.oden	2007-08-07 17:32:29.000000000 +0200
@@ -484,7 +484,7 @@
 
 	ZEND_FETCH_RESOURCE(session, LIBSSH2_SESSION*, &zsession, -1, PHP_SSH2_SESSION_RES_NAME, le_ssh2_session);
 
-#if LIBSSH2_APINO < 200412301450
+#if LIBSSH2_VERSION_NUM < 0x001000
 	libssh2_session_methods(session, &kex, &hostkey, &crypt_cs, &crypt_sc, &mac_cs, &mac_sc, &comp_cs, &comp_sc, &lang_cs, &lang_sc);
 #else
 	kex = libssh2_session_methods(session, LIBSSH2_METHOD_KEX);
